docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+### Style and configure the animation
+
+To illustrate these concepts, let's animate the journey of three birds from January to April of 2014 using data from [movebank.org](https://www.movebank.org/).
+
+#### Animate points
+
+At the beginning of the guide we created a simple point animation using the attribute `$date_time` for the temporal input (the date of the bird track and associated timestamp), set the duration to `10` seconds and both fade parameters to `1`. 
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1, 1))
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-1"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-1.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### Adjust the duration
+
+In the animation above, we can see that the birds spend the majority of this three month period in West Africa and then eventually migrate to Northern Europe. To see the journey in more detail, we'll adjust the duration to `30` seconds. Now looking at the map, we can see that each bird, at a different time, flies north along the coast of Morocco to Eastern Europe.
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 30, fade(1, 1))
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-2"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-2.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### Adjust fade
+
+Since we are visualizing  bird migration, we want to show more clearly, through symbology, the journey the birds took. We can get this added effect by adjusting  the`fade` parameters. By setting `fadeIn` to `0` and `fadeOut` to `0.5`, we  keep the previous point in the animation visible for longer which helps us better visualize the flight path of the migration journey.
+
+```js
+filter: animation($date_time, 30, fade(0, 0.5))
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-3"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-3.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### Final touches
+
+Since this visualization represents the journey of three different birds, we can assign  a unique color to  each one  using their names to define `buckets` inside of a `ramp`.decrease the `width` of the points to `4` and finally, remove the `strokeWidth` by setting it to `0`. 
+
+*Note:*
+For a more in-depth discussion of ramps and other styling properties, check out our [Data Driven Visualizations Guide](/developers/carto-vl/guides/0x-data-driven-visualizations.md). 
```

Comment:
Great!. We'll have to check this url later on, when https://github.com/CartoDB/carto-vl/pull/1005 will be integrated